tools/internal_ci/macos/pull_request/grpc_objc_bazel_test.cfg

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Config file for the internal CI (in protobuf text format)
+
+# Location of the continuous shell script in repository.
+build_file: "grpc/tools/internal_ci/macos/grpc_objc_bazel_test.sh"
+timeout_mins: 90
```

Comment:
might be good to reduce this down to e.g. 20 ~ 30min so that it fails fast in a bazel test

Reply:
we've had problems with timeouts for jobs being set too low in various scenarios (e.g. there's some wait to get a worker provisioned if there a queue etc), so I tend to set timeout_mins with some extra buffer to avoid problems.
Value of 90 is consistent with what we do for other jobs (some of which are also much faster then 90mins), so let's leave as is.
The question of how to set timeout_mins seems orthogonal to what's being setup in this PR and it also requires more though (and in my experience spending too much time trying to figure out the "right" value to use wasn't very productive in the past).